app/helpers/select_helper.rb

```diff
@@ -9,6 +17,23 @@ def array_to_select_collection(array, transform_func = :humanize)
     end
   end
 
+  # Returns roles which current user can set when creating/editing other users
```

Comment:
I would personally allow only super users to create accounts, and edit roles.

Reply:
Just to make things easier

Reply:
Ohh, I see that's was on the story and I guess decided with the client, so nevermind.